app/Models/EntityType.php

```diff
@@ -41,6 +41,7 @@ class EntityType extends Model
 
     public $fillable = [
         'id',
+        'campaign_id',
```

Comment:
We can't have campaign_id fillable or someone can change the module from one campaign to another